ideas/96-message-tributes.md

```diff
@@ -0,0 +1,110 @@
+
+If User A requires a minimum 200 SNT deposit, User B will not be able to open a new chat with User A until they deposit at least 200 SNT. Once deposited, User B can send their message.
+
+If User A responds to User B’s message, the deposit is transferred to User A, and a conversation is opened.
+
+If User A does not respond or User B changes their mind, User B can cancel or withdraw the request. The request will also time out automatically if User A does not respond in some number of days. In any case where User A does not respond, the deposit is returned to User B.
+
+If User A rejects User B’s request, User B’s deposit is returned. User B will have to wait some amount of time before they are able to send a new request to User A.
+
+In the future, User A might have the option to respond to User B but waive the deposit. User B might have the option to make a tribute to User A without requiring a response. 
+
+
+### Requirements & Dependencies
+<!-- Are there bugs or feature requests in other repositories that are part of this Idea? -->
+<!-- There is no approval unless the idea requires to be reviewed by supporting organelles (Financial, Hiring, or Design). -->
+<!-- The Swarm must develop a fully fleshed out Requirements document for the idea to proceed, to the satisfaction of participants. -->
+
+### Minimum Viable Product
+<!-- Mandatory, completes the Idea in the fastest route possible, can be hacky, needed to feel progress. See https://imgur.com/a/HVlw3 -->
+Goal Date: TBD <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+For the initial delivery, these are the minimum features proposed.
+- [ ] UI for recipients to set Tribute to Talk with required minimum deposit for any new chat
+- [ ] UI for requesters to send minimum SNT deposit and their message to users with Tribute to Talk enabled
+- [ ] UI for requesters to cancel a request and withdraw deposit
+- [ ] UI for recipient to respond to or reject new requests 
+- [ ] Deposit is transferred from requester to recipient when recipient responds
+- [ ] Deposit is returned to requester if rejected or withdrawn
+
+Next milestones and deliverables would be determined discussing and prioritizing features of the product description
+
+## Dates
+Goal Date: TBD <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: TBD <!-- Description of Deliverables-->
+
+Testing Days required: TBD <!-- Days required at the end of development for testing -->
+
+## Exit Criteria
+Any user can set up Tribute to Talk and require a deposit minimum to be contacted. Other users can put down a deposit to message them. New requests can be responded to, rejected or withdrawn. The deposit is transferred when a recipient responds, and sent back if they reject or the requester withdraws.
+
+## Success Metrics
```

Comment:
Would like to come to a team decision on which currencies we'll accept for deposits—just SNT, or both SNT and ETH. 

In either case, we're aiming with TtT to launch a working SNT use case (Q2 P2:P0 OKR). Based on this KR, success metrics focus on functionality rather than adoption.

Reply:
cc @hesterbruikman for review here